Runtimes/Supplemental/Readme.md

```diff
@@ -0,0 +1,52 @@
+# Swift Supplemental Libraries
+
+The supplemental libraries are all libraries that are not one of the Core or
+overlay libraries. Each supplemental library builds as an independent project.
+
+The supplemental libraries are:
+ - CxxInterop
+ - Differentiation
+ - Distributed
+ - Observation
+ - StringProcessing
+ - SwiftRuntime
```

Comment:
[Optional] I would prefer to refer to this with the name exposed to users, not the one we use in the build system

```suggestion
 - Runtime
```